src/ToDo.UI/Views/WelcomePage.xaml

```diff
@@ -1,28 +1,116 @@
-﻿<Page
-	x:Class="ToDo.Views.WelcomePage"
-	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-	xmlns:local="using:ToDo.Views"
-	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	xmlns:uen="using:Uno.Extensions.Navigation.UI"
-	xmlns:uer="using:Uno.Extensions.Reactive.UI"
-	xmlns:utu="using:Uno.Toolkit.UI"
-	xmlns:um="using:Uno.Material"
-	mc:Ignorable="d"
-	Background="{ThemeResource MaterialBackgroundBrush}">
+﻿<Page x:Class="ToDo.Views.WelcomePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:ToDo.Views"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:uen="using:Uno.Extensions.Navigation.UI"
+	  xmlns:uer="using:Uno.Extensions.Reactive.UI"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  xmlns:um="using:Uno.Material"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource MaterialBackgroundBrush}">
 
-	<utu:AutoLayout Padding="10,10,10,10">
-		<utu:AutoLayout Padding="20,60,20,60" Orientation="Horizontal" utu:AutoLayout.PrimaryAlignment="Stretch">
-			<Image Source="ms-appx:///Assets/Welcome/welcome_uno.png" Stretch="Uniform" utu:AutoLayout.CounterAlignment="Start" utu:AutoLayout.PrimaryAlignment="Stretch" utu:AutoLayout.CounterLength="310" />
+	<Grid>
+		<VisualStateManager.VisualStateGroups>
+			<VisualStateGroup>
+				<VisualState x:Name="Wide">
+					<VisualState.StateTriggers>
+						<AdaptiveTrigger MinWindowWidth="{StaticResource WideMinWindowWidthh}" />
```

Comment:
"WideMinWindowWidt**hh**" having a stroke?

Reply:
also where is the resource defined? it doesnt appear to be in uno or uwp

Reply:
@Xiaoy312 defined here, I will adjust the typo 
https://github.com/unoplatform/uno.todo/blob/aedcb706541602fd5648c599e7680aa79e238415/src/ToDo.UI/App.xaml#L39